src/cortxfs/cortxfs_ops.c

```diff
@@ -905,22 +905,71 @@ int cfs_rmdir(struct cfs_fs *cfs_fs, cfs_cred_t *cred,
 	return rc;
 }
 
+int cfs_unlink2(struct cfs_fh *parent_fh, struct cfs_fh *child_fh,
+                       cfs_cred_t *cred, char *name)
+{
+	int rc;
+
+	dassert(parent_fh && child_fh && cred && name);
+
+	RC_WRAP_LABEL(rc, out, cfs_detach2, parent_fh, child_fh, cred, name);
+	RC_WRAP_LABEL(rc, out, cfs_destroy_orphaned_file2, child_fh);
+
+out:
+	log_debug("parent_ino=%llu child_ino=%llu name=%s rc=%d",
```

Comment:
Can we also print the pointer to cfs_fs here. This would help us debugging in multi fs scenarios to uniquely identify a file.

Reply:
Done, addressed at 22e46940b661504ddfd7aeaec88f02b541cc9e56